PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp

```diff
@@ -2633,6 +2633,16 @@ void ScintillaEditView::performGlobalStyles()
 	execute(SCI_SETFOLDMARGINCOLOUR, true, foldMarginColor);
 	execute(SCI_SETFOLDMARGINHICOLOUR, true, foldMarginHiColor);
 
+	COLORREF bookmarkMarginColor = veryLiteGrey;

```

Comment:
discrete & elegant. Nice choice!

Reply:
Looking at it again 2 weeks later, to me it looks a bit _too_ discreet.
But, at least now user can change it to make it more contrasting if they'd like. :-)